qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -163,6 +165,19 @@ def _opt_control(property_set):
 
     _opt = [Optimize1qGates(basis_gates), CXCancellation()]
 
+    # 10. Schedule the circuit only when scheduling_method is supplied
+    if scheduling_method:
+        from qiskit.transpiler.passes import DelayInDt
+        _scheduling = [DelayInDt(instruction_durations.schedule_dt)]
```

Comment:
Isn't it strange that `InstructionDurations` stores `schedule_dt` for [telling it to `DelayInDt` pass](https://github.com/Qiskit/qiskit-terra/blob/cbe7ac22046c6ed28435c59e64fa13cd81714e96/qiskit/transpiler/preset_passmanagers/level0.py#L143)? I'm wondering if `schedule_dt` could be an optional argument of `tranpile()` and added to a member of `PassManagerConfig`. What do you think? @lcapelluto

Reply:
What is `schedule_dt`? Shouldn't this come straight from the backend?

Reply:
@ajavadia

Reply:
it should. I'm not a fan of DelayInDt as I mentioned above. It is fundamentally for translating circuit object to pulse object, and that's the job of the InstructionScheduleMap in my opinion.